_harp/programme/_overview.md

```diff
@@ -1,38 +1,32 @@
-We have graduated more than 150 students on our full-time programme. Over the last two years, more than 90% of our graduates have gone on to work in software or a related field.
+We have graduated more than 180 students on our full-time programmes in [London](./course-information/london) and [Nazareth](./course-information/nazareth). Over the last two years, more than 90% of our graduates have gone on to work in software or a related field.
 
-We believe that high quality education should be accessible to everyone. Our students continue to be involved with our programme long after they graduate. With their support and the work of the broader Founders & Coders community, we continue to offer our programme free at the point of delivery.
+We believe that high quality education should be accessible to everyone. Our students continue to be involved with our programme long after they graduate. With their support and the work of the broader Founders & Coders community, we continue to offer our programmes for free at the point of delivery.
 
-As of February 2017, we offer two programme locations: [London](/programme/course-information/london), UK and [Nazareth](/programme/course-information/nazareth), Israel.
+Founders & Coders welcomes applicants over 18 from all academic and professional backgrounds. We find that successful students are self-motivated, prepared to work long hours, and committed to collaborative working. Coding also requires a deep curiosity about how things work and a certain stubbornness in the face of repeated failure.
 
-### Pre-course introductory study
+### Self-learning and meetups
 
-Prospective students use online materials to teach themselves the fundamentals of programming and have the option to attend  weekly in-person meetups at the Founders & Coders campus while they are preparing and submitting their applications to the course.
+Prospective students use online materials to teach themselves the fundamentals of programming and have the option to attend weekly in-person meetups at the Founders & Coders campus while they are preparing and submitting their applications to the course.
 
-Visit [our Meetup site](http://www.meetup.com/founderscoders/) to learn more.
+### Prep-course
 
-### 16-week full-time coding bootcamp
+After conducting interviews, we select a group of finalists for an in-person, weekly prep-course, where we provide support completing [our course prerequisites](../apply/prerequisites).
 
```

Comment:
Maybe we should add a note explaining that this is new and that it's not planned for London until the Feb cohort?